MapboxCoreNavigation/MBXAccounts+CoreNavigationAdditions.m

```diff
@@ -0,0 +1,20 @@
+#import "MBXAccounts+CoreNavigationAdditions.h"
+
+static NSString * const MBXNavigationBillingMethodUser = @"user";
+static NSString * const MBXNavigationBillingMethodRequest = @"request";
+
+@implementation MBXAccounts (CoreNavigationAdditions)
+
++ (void)load {
+    NSString *billingMethodValue = [NSBundle.mainBundle objectForInfoDictionaryKey:@"MBXNavigationBillingMethod"];
+    if (!billingMethodValue.length || [billingMethodValue isEqualToString:MBXNavigationBillingMethodUser]) {
+        [MBXAccounts activateSKUID:MBXAccountsSKUIDNavigationUser];
+    } else if (![billingMethodValue isEqualToString:MBXNavigationBillingMethodRequest]) {
+        // Billing method is not the default in the absence of a SKU ID.
+        [NSException raise:@"MBXInvalidNavigationBillingMethod"
+                    format:@"Unrecognized billing method %@. Valid billing methods are: %@, %@.",
+         billingMethodValue, MBXNavigationBillingMethodUser, MBXNavigationBillingMethodRequest];
+    }
+}
+
+@end
```

Comment:
Do these have to be in OBJ-C?

Reply:
Yes, compiler simply does not permit to define such method in Swift.

Reply:
Here's a pretty nice workaround https://stackoverflow.com/a/47621197
It would be both Obj-C and Swift but at least the business logic could be written in Swift.
Maybe that could be cleaner to Swiftify `MBXNavigationBillingMethod`.

Reply:
It still produces some Obj-C code as well as a wrapper class and so on, so... Will we win something if we do it that way?

Reply:
The only benefit I see is that we could `switch` over `MBXNavigationBillingMethod` instead of using `isEqualToString:` if it became a Swift enum but I don't have a strong opinion for or against it.

Reply:
Agreed, `switch` would look much better. But it will introduce an implicit behavior: Obj-C wrapper class that calls "random" Swift class(es) static method which has to have a unique (to avoid possible collisions) name. It compiles slower, it runs slower, it takes more time to understand what is going on in such code. As I don't see benefits that outweigh these cons, I would prefer to just leave this method Obj-C.